Command/GitPersistenceAwareCommand.php

```diff
@@ -16,7 +16,7 @@
  *
  * @license http://opensource.org/licenses/MIT MIT License
  * @author Pascal Thormeier <pascal.thormeier@liip.ch>
- * @copyright Copyright (c) 2013, Liip, http://www.liip.ch
+ * @copyright Copyright (c) 2014, Liip, http://www.liip.ch
```

Comment:
well the file was added in 2014 .. in general it should then become a range .. ie. 2014-2015 .. but imho its all a bit BS and I never had any case where it mattered ..